### ISSUE TITLE
[terra-table] Checkbox does not read selection state change in chrome + jaws.

### ISSUE THREAD

**Issue by dkasper-was-taken** [cerner/terra-core#3044]
# Bug Report
Checkbox does not read selection state change in chrome + jaws.

## Description
When using JAWS w/ Chome the activation of a checkbox does not read the newly updated state. It will only read "space" instead of also announcing the new state.

## Steps to Reproduce
1. Launch terra-ui.com with Chrome, navigate to terra-table's "Check Mark Table" example.
2. Launch JAWS, navigation to checkbox element within table.
3. Use space to toggle the state of the checkbox.
4. Observe state change not read.

## Expected Behavior
JAWS should read the interaction and the new checkbox state.

## Environment
* Component Name and Version: terra-table v4.9.0
* Browser Name and Version: Chrome Version 83.0.4103.116
* Operating System and version (desktop or mobile): Windows 10

## @ Mentions
@scottwilmarth
